src/main.py

```diff
@@ -16,17 +16,32 @@
 from selenium.webdriver.chrome.service import Service
 
 # Load in the credentials from .env
-load_dotenv()
+load_dotenv(find_dotenv())
 
 # Set the email and password variables from the environment variables
 email = os.getenv('EMAIL')
 password = os.getenv('PASSWORD')
 token = os.getenv('TOKEN')
 newpassword = os.getenv('NEWPASSWORD')
 
-def totp(gen):
+def main(mode,code):
+	# Chromium oprions.
+	options = Options()
+	options.add_argument('--log-level=1')
+	options.add_experimental_option("excludeSwitches", ["enable-logging"])
+	# Sets what page to use.
+	match mode:
+		case "OTP":
+			# OTP login
+			dispage = "https://www.discord.com/login"
+		case "PR":
+			# Password reset page
+			options.add_argument('--headless')

```

Comment:
It might be wise if the `--headless` option is removed.

This is because the user must be able to stop and monitor the process at any time, for any purpose. A headless browser hides what's going on from the user, which is not ideal especially if the user is security and ip-ban-conscious.

Reply:
Made the option off by default with the use of a comment, so if users want to use `--headless` they just need to uncomment it.